students/Wayne/Assignments/Lesson2/FizzBuzzExercise.py

```diff
@@ -0,0 +1,12 @@
+#Creating FizzBuzz
+
+
+for i in range(1,101):
+    if i % 3 == 0 and not i % 5 == 0:
+        print('fizz')
+    elif i % 5 == 0 and not i % 3 == 0:
+        print('buzz')
+    elif i % 3 ==0 and i % 5 == 0:
```

Comment:
If you put this last elif higher up you could avoid the "and not" parts in your other two if/elifs.